pkg/infradb/subscriberframework/eventbus/event_bus_test.go

```diff
@@ -0,0 +1,181 @@
+package eventbus
+
+import (
+	"log"
+	"sync"
+	"testing"
+	"time"
+)
+
+type ModulelciHandler struct {
+	receivedEvents []*ObjectData
+	sync.Mutex
+}
+
+func (h *ModulelciHandler) HandleEvent(eventType string, objectData *ObjectData) {
+	h.Lock()
+	defer h.Unlock()
+	h.receivedEvents = append(h.receivedEvents, objectData)
+	switch eventType {
+	case "testEvent":
```

Comment:
why not create constants for those values? It would be much harder to make a mistake by a typo

Reply:
line 101
```go
func (h *moduleCiHandler) HandleEvent(eventType string, objectData *eventbus.ObjectData) {
	h.receivedEvents = append(h.receivedEvents, objectData)
	switch eventType {
	case "testEvent":
		handleTestEvent(objectData)
	case "testEventCompSuccess":
		handleTestEventCompSuccess(objectData)
	case "testEventError":
		handleTestEventError(objectData)
	case "testEventTimeout":
		handletestEventTimeout()
	case "testEventNotificationIdNotMatching":
		handletestNotificationIDNotMatching(objectData)
	default:
		log.Printf("LCI: error: Unknown event type %s", eventType)
	}
}
```